playbooks/common/openshift-node/config.yml

```diff
@@ -123,21 +123,21 @@
   - os_env_extras
   - os_env_extras_node
 
-- name: Set scheduleability
+- name: Set schedulability
   hosts: oo_first_master
   vars:
     openshift_nodes: "{{ hostvars
                          | oo_select_keys(groups['oo_nodes_to_config'])
                          | oo_collect('openshift.common.hostname') }}"
-    openshift_unscheduleable_nodes: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config'] | default([]))
-                                      | oo_collect('openshift.common.hostname', {'openshift_scheduleable': False}) }}"
+    openshift_unschedulable_nodes: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config'] | default([]))
+                                      | oo_collect('openshift.common.hostname', {'openshift_schedulable': False}) }}"
     openshift_node_vars: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config']) }}"
   pre_tasks:
   - set_fact:
-      openshift_scheduleable_nodes: "{{ hostvars
+      openshift_schedulable_nodes: "{{ hostvars
                                       | oo_select_keys(groups['oo_nodes_to_config'] | default([]))
                                       | oo_collect('openshift.common.hostname')
-                                      | difference(openshift_unscheduleable_nodes) }}"
+                                      | difference(openshift_unschedulable_nodes) }}"
```

Comment:
This set_fact wouldn't be needed with moving the schedulable fact under the node.